### DISCUSSION TITLE
How to ignore mouse events on top so control underneath can deal with it?

### DISCUSSION THREAD

**Discussion by sonicsmooth** [AvaloniaUI/Avalonia#10638] [General]
I have a `Panel` which is drawn over (aka after) a `TemplatedControl`.
They are both siblings under the MainWindow parent.
The `TemplatedControl` has handlers for mouse click and drag, etc., and holds the render transform for both items.
How do I make the `Panel` *not* capture the mouse movement and let the `TemplatedControl` handle it instead?

I have tried setting `handledEventsToo` in the `Panel`'s codebehind without success:

    this.AddHandler(PointerMovedEvent, OnPointerMoved, handledEventsToo: true);

thanks

**Comment by maxkatz6** [comment]
> How do I make the Panel not capture the mouse movement and let the TemplatedControl handle it instead

Your Panel won't handle mouse capture unless you specifically make it to do so. Or put other controls that would do so (like, Buttons).
If you want to make whole control invisible for the hit test, then set IsHitTestVisible to false.
